mpc-core/src/garble/exec/semihonest.rs

```diff
@@ -220,44 +228,19 @@ mod tests {
 
         let (input_labels, delta) = InputLabels::generate(&mut rng, &circ, None);
 
-        let (mut msg, _) = leader
+        let (mut gc_partial, _) = leader
             .garble(&[leader_input], &input_labels, delta, true)
             .unwrap();
 
         // Insert bogus output label commitments
-        msg.commitments.as_mut().unwrap()[0].commitments[0] = Block::new(0);
-        msg.commitments.as_mut().unwrap()[0].commitments[1] = Block::new(1);
+        gc_partial.state.commitments.as_mut().unwrap()[0].commitments[0][0] = Block::new(0);
+        gc_partial.state.commitments.as_mut().unwrap()[0].commitments[0][1] = Block::new(1);
 
         let follower_labels = input_labels[1].select(&follower_input).unwrap();
-        let error = follower.evaluate(msg, &[follower_labels]).unwrap_err();
+        let error = follower
+            .evaluate(gc_partial.into(), &[follower_labels])
+            .unwrap_err();
 
         assert!(matches!(error, Error::InvalidOutputLabelCommitment));
     }
-
```

Comment:
Why was this removed?

Reply:
I can't find where. Could you point to that place?

Reply:
I see what you mean. Then this test can be considered a test of that check.  A test can always prevent a future refactor error, so I don't see a reason to remove it.